R/default-scoring-rules.R

```diff
@@ -124,13 +124,47 @@ get_metrics.forecast_nominal <- function(x, select = NULL, exclude = NULL, ...)
 #' - "ae_point" = [ae()][Metrics::ae()]
 #' - "se_point" = [se()][Metrics::se()]
 #' - "ape" = [ape()][Metrics::ape()]
+#'
+#' A note of caution: Every scoring rule for a point forecast
+#' is implicitly minimised by a specific aspect of the predictive distribution
+#' (see Gneiting, 2011).
+#'
+#' The mean squared error, for example, is only a meaningful scoring rule if
+#' the forecaster actually reported the mean of their predictive distribution
+#' as a point forecast. If the forecaster reported the median, then the mean
+#' absolute error would be the appropriate scoring rule. If the scoring rule
+#' and the predictive task do not align, misleading results ensue.
```

Comment:
```suggestion
#' and the predictive task do not align, the results will be misleading.
```

spelling mistake but also sounded a bit weird